persistence/pre_persistence/app_test.go

```diff
@@ -115,19 +115,18 @@ func TestUpdateApp(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	before, err := ctx.(*PrePersistenceContext).GetApp(actor.Address, height)
-	require.NoError(t, err)
-	tokens := before.StakedTokens
-	bigBeforeTokens, err := types.StringToBigInt(tokens)
+	//before, err := ctx.(*PrePersistenceContext).GetApp(actor.Address, height)
```

Comment:
Add back or remove?

Just leaving one comment but there are a few commented out test sections, and not sure whether we need to keep them in.